airbyte-integrations/connectors/destination-clickhouse/src/test-integration/java/io/airbyte/integrations/destination/clickhouse/ClickhouseTestSourceOperations.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.clickhouse;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.airbyte.db.DataTypeUtils;
+import io.airbyte.db.jdbc.JdbcSourceOperations;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class ClickhouseTestSourceOperations extends JdbcSourceOperations {
+
+  public static final DateTimeFormatter CLICKHOUSE_DATETIME_WITH_TZ_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSX");
```

Comment:
do we use this constant?

Reply:
Yup, it's used in parseDestinationDateWithTz and compareDateTimeValues methods

Reply:
You were right, It didn't. Updated, thanks